runtime/j9vm/javanextvmi.cpp

```diff
@@ -793,10 +793,23 @@ JVM_GetCDSConfigStatus()
 #endif /* JAVA_SPEC_VERSION >= 23 */
 
 #if JAVA_SPEC_VERSION >= 24
+/**
+ * @brief Determine if the JVM is running inside a container.
+ *
+ * @return JNI_TRUE if running inside a container; otherwise, JNI_FALSE
+ */
 JNIEXPORT jboolean JNICALL
 JVM_IsContainerized(void)
 {
-	return JNI_FALSE;
+	J9JavaVM *vm = BFUjavaVM;
+	jboolean isContainerized = JNI_FALSE;
+	if (NULL != vm) {
```

Comment:
I was looking at other, similar functions that depend upon access to the VM; and noticed that `JVM_IsUseContainerSupport()` has the wrong signature - it gets no arguments contrary to the implementation in `java11vmi.c`:
```c
JNIEXPORT jboolean JNICALL
JVM_IsUseContainerSupport(JNIEnv *env)
```
I suggest that both should effectively `assert NULL != BFUjavaVM;`. Could you please do that here, and fix `JVM_IsUseContainerSupport()` to do likewise?

Reply:
Updated.